host/sampi/http.go

```diff
@@ -171,17 +199,21 @@ func (c *HTTPAPI) RemoveJob(w http.ResponseWriter, r *http.Request, ps httproute
 }
 
 func (c *HTTPAPI) StreamHostEvents(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	l := c.logger.New("fn", "StreamHostEvents")
```

Comment:
log lines from http requests should have details from the request in them (client address, request ID?)

It would make sense to set up a logger in a middleware.